src/GuppyDataExplorer/ExplorerButtonGroup/index.jsx

```diff
@@ -799,8 +800,8 @@ Currently, in order to export a File PFB, \`enableLimitedFilePFBExport\` must be
     return (
       <Button
         key={buttonConfig.type}
-        onClick={clickFunc}
-        label={buttonTitle}
+        onClick={() => { if (!this.isLoginForButtonFeatureEnabled() && this.isDownloadButton(buttonConfig)) { clickFunc(); } }}
+        label={(this.props.user.username && this.isDownloadButton(buttonConfig)) ? buttonTitle : `Login to ${buttonTitle}`}
```

Comment:
`buttonTitle` usually have first char as caps, so `Login to ${buttonTitle}` will looks like `Login to Download ...`
```suggestion
        label={(this.props.user.username && this.isDownloadButton(buttonConfig)) ? buttonTitle : `Login to ${buttonTitle.toLowerCase()}`}
```

Reply:
Updated this.